BUILD: No MariaDB Connector/C! build error despite installed submodule

### Make sure you've done the following:

- [X] I have read the [installation guide](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/README.md).


### DarkflameServer Version

008f953

### Platform

Ubuntu

### Architecture

x86

### Error Logs
This is an error with the Docker Build. I have followed the [Docker.md](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/Docker.md) file and installed all required submodules.

The error states `No MariaDB Connector/C!` despite submodule being installed.

This is my first issue, any redirection is appreciated.

## Error Message
<details>
#0 39.04 [ 19%] Performing configure step for 'mariadb_connector_cpp'
#0 39.11 -- The CXX compiler identification is GNU 11.3.0
#0 39.16 -- The C compiler identification is GNU 11.3.0
#0 39.17 -- Detecting CXX compiler ABI info
#0 39.26 -- Detecting CXX compiler ABI info - done
#0 39.27 -- Check for working CXX compiler: /usr/local/bin/c++ - skipped
#0 39.28 -- Detecting CXX compile features
#0 39.28 -- Detecting CXX compile features - done
#0 39.28 -- Detecting C compiler ABI info
#0 39.34 -- Detecting C compiler ABI info - done
#0 39.36 -- Check for working C compiler: /usr/local/bin/gcc - skipped
#0 39.36 -- Detecting C compile features
#0 39.36 -- Detecting C compile features - done
#0 39.36 -- Looking for floor
#0 39.43 -- Looking for floor - not found
#0 39.43 -- Looking for floor in m
#0 39.50 -- Looking for floor in m - found
#0 39.50 -- Found math lib: m
#0 39.51 -- MariaDB Connector ODBC: INSTALL_BINDIR=bin
#0 39.51 -- MariaDB Connector ODBC: INSTALL_LIBDIR=lib/mariadb
#0 39.51 -- MariaDB Connector ODBC: INSTALL_PCDIR=lib/pkgconfig
#0 39.51 -- MariaDB Connector ODBC: INSTALL_INCLUDEDIR=include/mariadb
#0 39.51 -- MariaDB Connector ODBC: INSTALL_DOCDIR=share/doc/
#0 39.51 -- MariaDB Connector ODBC: INSTALL_LICENSEDIR=share/doc/
#0 39.51 -- MariaDB Connector ODBC: INSTALL_PLUGINDIR=plugin
#0 39.51 -- Libraries installation dir: lib
#0 39.51 -- Authentication Plugins installation dir: plugin
#0 39.51 -- Setting default value for WITH_SSL for libmariadb build to ON
#0 39.51 -- Running C/C cmake scripts
#0 39.52 -- Found Git: /usr/bin/git (found version "2.30.2")
#0 39.54 fatal: not a git repository: /build/thirdparty/mariadb-connector-cpp/../../.git/modules/thirdparty/mariadb-connector-cpp
#0 39.55 fatal: not a git repository: /build/thirdparty/mariadb-connector-cpp/../../.git/modules/thirdparty/mariadb-connector-cpp
#0 39.55 CMake Error at cmake/connector_c.cmake:10 (MESSAGE):
#0 39.55   No MariaDB Connector/C! Run
#0 39.55
#0 39.55       git submodule init
#0 39.55       git submodule update
#0 39.55
#0 39.55   Then restart the build.
#0 39.55
#0 39.55 Call Stack (most recent call first):
#0 39.55   CMakeLists.txt:391 (INCLUDE)
#0 39.55
#0 39.55
#0 39.55 -- Configuring incomplete, errors occurred!
#0 39.55 See also "/build/cmake_build/mariadbcpp/src/mariadb_connector_cpp-build/CMakeFiles/CMakeOutput.log".
#0 39.55 See also "/build/cmake_build/mariadbcpp/src/mariadb_connector_cpp-build/CMakeFiles/CMakeError.log".
#0 39.55 make[2]: *** [thirdparty/CMakeFiles/mariadb_connector_cpp.dir/build.make:125: mariadbcpp/src/mariadb_connector_cpp-stamp/mariadb_connector_cpp-configure] Error 1
#0 39.55 make[1]: *** [CMakeFiles/Makefile2:1551: thirdparty/CMakeFiles/mariadb_connector_cpp.dir/all] Error 2
#0 39.55 make: *** [Makefile:114: all] Error 2
------
failed to solve: executor failed running [/bin/sh -c echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_build &&     sed -i -e "s/171022/${BUILD_VERSION}/g" ../CMakeVariables.txt &&     cmake .. -DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE &&     make -j $BUILD_THREADS]: exit code: 2
</details>

## Submodule Installation
![image](https://user-images.githubusercontent.com/22532991/184518142-95334e98-7e85-46f4-ad26-820990ca05d8.png)


## Full Logs
<details>
emmett@pinecone:~/data/LegoUniverse/DarkflameServer$ docker compose up -d --build
[+] Building 41.1s (70/77)
 => [darkflameserver_account-manager internal] load build definition from AccountManager.Dockerfile                             0.0s
 => => transferring dockerfile: 402B                                                                                            0.0s
 => [darkflameserver_setup internal] load build definition from setup.Dockerfile                                                0.0s
 => => transferring dockerfile: 560B                                                                                            0.0s
 => [darkflameserver_database internal] load build definition from database.Dockerfile                                          0.0s
 => => transferring dockerfile: 114B                                                                                            0.0s
 => [darkflameserver_darkflame internal] load build definition from Dockerfile                                                  0.0s
 => => transferring dockerfile: 1.84kB                                                                                          0.0s
 => [darkflameserver_brickbuildfix internal] load build definition from brickfix.Dockerfile                                     0.0s
 => => transferring dockerfile: 210B                                                                                            0.0s
 => [darkflameserver_account-manager internal] load .dockerignore                                                               0.0s
 => => transferring context: 34B                                                                                                0.0s
 => [darkflameserver_setup internal] load .dockerignore                                                                         0.0s
 => => transferring context: 34B                                                                                                0.0s
 => [darkflameserver_database internal] load .dockerignore                                                                      0.0s
 => => transferring context: 34B                                                                                                0.0s
 => [darkflameserver_darkflame internal] load .dockerignore                                                                     0.0s
 => => transferring context: 34B                                                                                                0.0s
 => [darkflameserver_brickbuildfix internal] load .dockerignore                                                                 0.0s
 => => transferring context: 34B                                                                                                0.0s
 => [darkflameserver_account-manager internal] load metadata for docker.io/library/python:3.10-alpine3.14                       1.4s
 => [darkflameserver_setup internal] load metadata for docker.io/library/rust:alpine3.14                                        1.1s
 => [darkflameserver_database internal] load metadata for docker.io/library/mariadb:10.6                                        1.1s
 => [darkflameserver_darkflame internal] load metadata for docker.io/library/gcc:11                                             1.1s
 => [darkflameserver_account-manager] resolve image config for docker.io/docker/dockerfile:1                                    1.2s
 => [darkflameserver_database internal] load build context                                                                      0.0s
 => => transferring context: 5.62kB                                                                                             0.0s
 => [darkflameserver_database 1/2] FROM docker.io/library/mariadb:10.6@sha256:e78a7000d93b2fabc0bfb6ff1504199f2f9bfb4a8b735092  0.0s
 => [darkflameserver_darkflame internal] load build context                                                                     0.1s
 => => transferring context: 143.38kB                                                                                           0.1s
 => [darkflameserver_darkflame build  1/24] FROM docker.io/library/gcc:11@sha256:32f7156e3bfad8a61470e8f0081360ab85d17af7e9037  0.0s
 => CACHED [darkflameserver_darkflame runtime 2/6] RUN --mount=type=cache,id=runtime-apt-cache,target=/var/cache/apt     apt u  0.0s
 => CACHED [darkflameserver_darkflame runtime 3/6] WORKDIR /app                                                                 0.0s
 => CACHED [darkflameserver_database 2/2] COPY ./migrations/dlu /docker-entrypoint-initdb.d                                     0.0s
 => [darkflameserver_account-manager] exporting to image                                                                        0.0s
 => => exporting layers                                                                                                         0.0s
 => => writing image sha256:11bbebc7f9c53e83cfdeee9f2f6ebb56ea6a9bd1b844ada3ed16db208165e4f9                                    0.0s
 => => naming to docker.io/library/darkflameserver_database                                                                     0.0s
 => => writing image sha256:6603216a8b64894183a51105d9fbb1690b239d925ed22cb9c26fa07176bc9098                                    0.0s
 => => naming to docker.io/library/darkflameserver_brickbuildfix                                                                0.0s
 => => writing image sha256:6e6d0840a107d273d7707882e95e62058ec0fb0ce059d8f67452ef1e9f7df2f5                                    0.0s
 => => naming to docker.io/library/darkflameserver_account-manager                                                              0.0s
 => CACHED [darkflameserver_brickbuildfix] docker-image://docker.io/docker/dockerfile:1@sha256:443aab4ca21183e069e7d8b2dc68006  0.0s
 => [darkflameserver_account-manager prep 1/8] FROM docker.io/library/python:3.10-alpine3.14@sha256:fb93ca595ad82020cc52ff6060  0.0s
 => [darkflameserver_setup internal] load build context                                                                         0.1s
 => => transferring context: 7.27MB                                                                                             0.1s
 => [darkflameserver_setup lunpack 1/4] FROM docker.io/library/rust:alpine3.14@sha256:2e07fc0d66b00d6acc4b243f65f292aa21afc8a1  0.0s
 => CACHED [darkflameserver_darkflame build  2/24] WORKDIR /build                                                               0.0s
 => CACHED [darkflameserver_darkflame build  3/24] RUN --mount=type=cache,id=build-apt-cache,target=/var/cache/apt     echo "I  0.0s
 => CACHED [darkflameserver_darkflame build  4/24] COPY dAuthServer/ /build/dAuthServer                                         0.0s
 => CACHED [darkflameserver_darkflame build  5/24] COPY dChatServer/ /build/dChatServer                                         0.0s
 => CACHED [darkflameserver_darkflame build  6/24] COPY dCommon/ /build/dCommon                                                 0.0s
 => CACHED [darkflameserver_darkflame build  7/24] COPY dChatFilter/ /build/dChatFilter                                         0.0s
 => CACHED [darkflameserver_darkflame build  8/24] COPY dDatabase/ /build/dDatabase                                             0.0s
 => CACHED [darkflameserver_darkflame build  9/24] COPY dGame/ /build/dGame                                                     0.0s
 => CACHED [darkflameserver_darkflame build 10/24] COPY dMasterServer/ /build/dMasterServer                                     0.0s
 => CACHED [darkflameserver_darkflame build 11/24] COPY dNet/ /build/dNet                                                       0.0s
 => CACHED [darkflameserver_darkflame build 12/24] COPY dPhysics/ /build/dPhysics                                               0.0s
 => CACHED [darkflameserver_darkflame build 13/24] COPY dScripts/ /build/dScripts                                               0.0s
 => CACHED [darkflameserver_darkflame build 14/24] COPY dWorldServer/ /build/dWorldServer                                       0.0s
 => CACHED [darkflameserver_darkflame build 15/24] COPY dZoneManager/ /build/dZoneManager                                       0.0s
 => CACHED [darkflameserver_darkflame build 16/24] COPY dNavigation/ /build/dNavigation                                         0.0s
 => CACHED [darkflameserver_darkflame build 17/24] COPY migrations/ /build/migrations                                           0.0s
 => CACHED [darkflameserver_darkflame build 18/24] COPY resources/ /build/resources                                             0.0s
 => CACHED [darkflameserver_darkflame build 19/24] COPY thirdparty/ /build/thirdparty                                           0.0s
 => CACHED [darkflameserver_darkflame build 20/24] COPY vanity /build/vanity                                                    0.0s
 => CACHED [darkflameserver_darkflame build 21/24] COPY tests/ /build/tests                                                     0.0s
 => CACHED [darkflameserver_darkflame build 22/24] COPY .clang-* CMake* LICENSE /build/                                         0.0s
 => ERROR [darkflameserver_darkflame build 23/24] RUN echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_build &  39.6s
 => CACHED [darkflameserver_setup lunpack 2/4] WORKDIR /build_LUnpack                                                           0.0s
 => CACHED [darkflameserver_setup lunpack 3/4] COPY ./thirdparty/LUnpack .                                                      0.0s
 => CANCELED [darkflameserver_setup lunpack 4/4] RUN apk add musl-dev --no-cache && cargo build --release                      39.7s
 => CACHED [darkflameserver_setup prep 2/8] RUN apk add sqlite bash --no-cache                                                  0.0s
 => CACHED [darkflameserver_setup prep 3/8] WORKDIR /setup                                                                      0.0s
 => CACHED [darkflameserver_setup prep 4/8] COPY resources/ resources/                                                          0.0s
 => CACHED [darkflameserver_setup prep 5/8] COPY migrations/cdserver/ migrations/cdserver                                       0.0s
 => [darkflameserver_brickbuildfix internal] load .dockerignore                                                                 0.0s
 => [darkflameserver_brickbuildfix internal] load build definition from brickfix.Dockerfile                                     0.0s
 => [darkflameserver_brickbuildfix internal] load metadata for docker.io/library/python:3.9.9-slim                              0.1s
 => [darkflameserver_account-manager internal] load .dockerignore                                                               0.0s
 => [darkflameserver_account-manager internal] load build definition from AccountManager.Dockerfile                             0.0s
 => [darkflameserver_brickbuildfix 1/3] FROM docker.io/library/python:3.9.9-slim@sha256:f4efbe5d1eb52c221fded79ddf18e4baa0606e  0.0s
 => CACHED [darkflameserver_brickbuildfix 2/3] RUN apt-get update &&     apt-get install curl -y                                0.0s
 => CACHED [darkflameserver_brickbuildfix 3/3] WORKDIR /empty_dir                                                               0.0s
 => [darkflameserver_account-manager internal] load build context                                                               0.0s
 => => transferring context: 1.31kB                                                                                             0.0s
 => CACHED [darkflameserver_account-manager 2/6] WORKDIR /usr/local/share/AccountManager                                        0.0s
 => CACHED [darkflameserver_account-manager 3/6] COPY ./thirdparty/AccountManager .                                             0.0s
 => CACHED [darkflameserver_account-manager 4/6] ADD docker/credentials_example.py credentials.py                               0.0s
 => CACHED [darkflameserver_account-manager 5/6] ADD docker/resources_example.py resources.py                                   0.0s
 => CACHED [darkflameserver_account-manager 6/6] RUN apk add curl libffi-dev build-base --no-cache && pip3 install -r requirem  0.0s
------
 > [darkflameserver_darkflame build 23/24] RUN echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_build &&     sed -i -e "s/171022/171022/g" ../CMakeVariables.txt &&     cmake .. -DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE &&     make -j 1:
#0 0.329 Build server
#0 0.492 -- The C compiler identification is GNU 11.3.0
#0 0.595 -- The CXX compiler identification is GNU 11.3.0
#0 0.607 -- Detecting C compiler ABI info
#0 0.671 -- Detecting C compiler ABI info - done
#0 0.682 -- Check for working C compiler: /usr/bin/cc - skipped
#0 0.682 -- Detecting C compile features
#0 0.683 -- Detecting C compile features - done
#0 0.687 -- Detecting CXX compiler ABI info
#0 0.771 -- Detecting CXX compiler ABI info - done
#0 0.784 -- Check for working CXX compiler: /usr/local/bin/c++ - skipped
#0 0.785 -- Detecting CXX compile features
#0 0.785 -- Detecting CXX compile features - done
#0 0.790 -- Variable: PROJECT_VERSION_MAJOR = 1
#0 0.790 -- Variable: PROJECT_VERSION_MINOR = 0
#0 0.790 -- Variable: PROJECT_VERSION_PATCH = 4
#0 0.790 -- Variable: LICENSE = AGPL-3.0
#0 0.790 -- Variable: NET_VERSION = 171022
#0 0.790 -- Variable: __dynamic = 1
#0 0.790 -- Variable: __maria_db_connector_compile_jobs__ = 1
#0 0.790 -- Version: 1.0.4
#0 0.791 Moved authconfig.ini to project binary directory
#0 0.791 Moved chatconfig.ini to project binary directory
#0 0.791 Moved worldconfig.ini to project binary directory
#0 0.791 Moved masterconfig.ini to project binary directory
#0 0.792 Moved blacklist.dcf to project binary directory
#0 0.827 -- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11")
#0 0.857 -- Configuring done
#0 0.947 -- Generating done
#0 0.951 -- Build files have been written to: /build/cmake_build
#0 1.014 Scanning dependencies of target sqlite3
#0 1.024 [  0%] Building CXX object thirdparty/CMakeFiles/sqlite3.dir/SQLite/CppSQLite3.cpp.o
#0 1.511 [  0%] Building C object thirdparty/CMakeFiles/sqlite3.dir/SQLite/sqlite3.c.o
#0 14.57 [  0%] Linking CXX static library ../libsqlite3.a
#0 14.61 [  0%] Built target sqlite3
#0 14.62 Scanning dependencies of target libbcrypt
#0 14.63 [  0%] Building C object thirdparty/CMakeFiles/libbcrypt.dir/libbcrypt/src/bcrypt.c.o
#0 14.68 [  0%] Building C object thirdparty/CMakeFiles/libbcrypt.dir/libbcrypt/src/crypt_blowfish.c.o
#0 14.97 [  1%] Building C object thirdparty/CMakeFiles/libbcrypt.dir/libbcrypt/src/crypt_gensalt.c.o
#0 15.02 [  1%] Building C object thirdparty/CMakeFiles/libbcrypt.dir/libbcrypt/src/wrapper.c.o
#0 15.07 [  1%] Linking C static library ../liblibbcrypt.a
#0 15.10 [  1%] Built target libbcrypt
#0 15.12 Scanning dependencies of target detour
#0 15.13 [  1%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourAlloc.cpp.o
#0 15.16 [  1%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourAssert.cpp.o
#0 15.20 [  1%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourCommon.cpp.o
#0 15.44 [  1%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourNavMesh.cpp.o
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp: In member function 'int dtNavMesh::findConnectingPolys(const float*, const float*, const dtMeshTile*, int, dtPolyRef*, float*, int) const':
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:46:34: warning: 'bmax[1]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    46 |         const float bd = (bmax[1]-bmin[1]) / (bmax[0]-bmin[0]);
#0 15.76       |                          ~~~~~~~~^~~~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:311:24: note: 'bmax[1]' was declared here
#0 15.76   311 |         float bmin[2], bmax[2];
#0 15.76       |                        ^~~~
#0 15.76 In file included from /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:24:
#0 15.76 /build/thirdparty/recastnavigation/Detour/Include/DetourCommon.h:52:59: warning: 'bmax[0]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    52 | template<class T> inline T dtMin(T a, T b) { return a < b ? a : b; }
#0 15.76       |                                                     ~~~~~~^~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:311:24: note: 'bmax[0]' was declared here
#0 15.76   311 |         float bmin[2], bmax[2];
#0 15.76       |                        ^~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:46:34: warning: 'bmin[1]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    46 |         const float bd = (bmax[1]-bmin[1]) / (bmax[0]-bmin[0]);
#0 15.76       |                          ~~~~~~~~^~~~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:311:15: note: 'bmin[1]' was declared here
#0 15.76   311 |         float bmin[2], bmax[2];
#0 15.76       |               ^~~~
#0 15.76 In file included from /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:24:
#0 15.76 /build/thirdparty/recastnavigation/Detour/Include/DetourCommon.h:58:59: warning: 'bmin[0]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    58 | template<class T> inline T dtMax(T a, T b) { return a > b ? a : b; }
#0 15.76       |                                                     ~~~~~~^~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:311:15: note: 'bmin[0]' was declared here
#0 15.76   311 |         float bmin[2], bmax[2];
#0 15.76       |               ^~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:44:34: warning: 'amax[1]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    44 |         const float ad = (amax[1]-amin[1]) / (amax[0]-amin[0]);
#0 15.76       |                          ~~~~~~~~^~~~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:306:24: note: 'amax[1]' was declared here
#0 15.76   306 |         float amin[2], amax[2];
#0 15.76       |                        ^~~~
#0 15.76 In file included from /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:24:
#0 15.76 /build/thirdparty/recastnavigation/Detour/Include/DetourCommon.h:52:59: warning: 'amax[0]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    52 | template<class T> inline T dtMin(T a, T b) { return a < b ? a : b; }
#0 15.76       |                                                     ~~~~~~^~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:306:24: note: 'amax[0]' was declared here
#0 15.76   306 |         float amin[2], amax[2];
#0 15.76       |                        ^~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:45:21: warning: 'amin[1]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    45 |         const float ak = amin[1] - ad*amin[0];
#0 15.76       |                     ^~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:306:15: note: 'amin[1]' was declared here
#0 15.76   306 |         float amin[2], amax[2];
#0 15.76       |               ^~~~
#0 15.76 In file included from /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:24:
#0 15.76 /build/thirdparty/recastnavigation/Detour/Include/DetourCommon.h:58:59: warning: 'amin[0]' may be used uninitialized in this function [-Wmaybe-uninitialized]
#0 15.76    58 | template<class T> inline T dtMax(T a, T b) { return a > b ? a : b; }
#0 15.76       |                                                     ~~~~~~^~~~~~~
#0 15.76 /build/thirdparty/recastnavigation/Detour/Source/DetourNavMesh.cpp:306:15: note: 'amin[0]' was declared here
#0 15.76   306 |         float amin[2], amax[2];
#0 15.76       |               ^~~~
#0 15.92 [  2%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourNavMeshBuilder.cpp.o
#0 16.28 [  2%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourNavMeshQuery.cpp.o
#0 17.22 [  2%] Building CXX object thirdparty/CMakeFiles/detour.dir/recastnavigation/Detour/Source/DetourNode.cpp.o
#0 17.41 [  2%] Linking CXX static library ../libdetour.a
#0 17.44 [  2%] Built target detour
#0 17.46 Scanning dependencies of target recast
#0 17.47 [  2%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/Recast.cpp.o
#0 17.76 [  2%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastAlloc.cpp.o
#0 17.81 [  2%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastArea.cpp.o
#0 18.14 [  2%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastAssert.cpp.o
#0 18.17 [  2%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastContour.cpp.o
#0 18.79 [  3%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastFilter.cpp.o
#0 19.00 [  3%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastLayers.cpp.o
#0 19.32 [  3%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastMesh.cpp.o
#0 19.94 [  3%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastMeshDetail.cpp.o
#0 20.67 [  3%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastRasterization.cpp.o
#0 20.94 [  3%] Building CXX object thirdparty/CMakeFiles/recast.dir/recastnavigation/Recast/Source/RecastRegion.cpp.o
#0 21.92 [  4%] Linking CXX static library ../librecast.a
#0 21.96 [  4%] Built target recast
#0 21.97 Scanning dependencies of target tinyxml2
#0 21.98 [  4%] Building CXX object thirdparty/CMakeFiles/tinyxml2.dir/cmake_pch.hxx.gch
#0 22.06 [  4%] Building CXX object thirdparty/CMakeFiles/tinyxml2.dir/tinyxml2/tinyxml2.cpp.o
#0 22.82 [  5%] Linking CXX static library ../libtinyxml2.a
#0 22.85 [  5%] Built target tinyxml2
#0 22.92 Scanning dependencies of target raknet
#0 22.93 [  5%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/AsynchronousFileIO.cpp.o
#0 22.95 [  5%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/AutoRPC.cpp.o
#0 23.38 [  6%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/BitStream.cpp.o
#0 23.66 [  6%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/BitStream_NoTemplate.cpp.o
#0 23.68 [  6%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/CheckSum.cpp.o
#0 23.73 [  6%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/CommandParserInterface.cpp.o
#0 23.84 [  6%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ConnectionGraph.cpp.o
#0 24.49 [  6%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ConsoleServer.cpp.o
#0 24.62 [  7%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DS_BytePool.cpp.o
#0 24.77 [  7%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DS_ByteQueue.cpp.o
#0 24.83 [  7%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DS_HuffmanEncodingTree.cpp.o
#0 25.07 [  7%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DS_Table.cpp.o
#0 25.56 [  7%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DataBlockEncryptor.cpp.o
#0 25.63 [  7%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DataCompressor.cpp.o
#0 25.81 [  8%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/DirectoryDeltaTransfer.cpp.o
#0 26.03 [  8%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/EmailSender.cpp.o
#0 26.29 [  8%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/EncodeClassName.cpp.o
#0 26.31 [  8%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/EpochTimeToString.cpp.o
#0 26.35 [  8%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ExtendedOverlappedPool.cpp.o
#0 26.37 [  8%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/FileList.cpp.o
#0 26.71 [  9%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/FileListTransfer.cpp.o
#0 27.06 [  9%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/FileOperations.cpp.o
#0 27.12 [  9%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/FormatString.cpp.o
#0 27.16 [  9%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/FullyConnectedMesh.cpp.o
#0 27.34 [  9%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/FunctionThread.cpp.o
#0 27.49 [  9%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/Gen_RPC8.cpp.o
#0 27.67 [ 10%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/GetTime.cpp.o
#0 27.71 [ 10%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/GridSectorizer.cpp.o
#0 27.90 [ 10%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/HTTPConnection.cpp.o
#0 28.03 [ 10%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/InlineFunctor.cpp.o
#0 28.10 [ 10%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/Itoa.cpp.o
#0 28.13 [ 10%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/LightweightDatabaseClient.cpp.o
#0 28.36 [ 11%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/LightweightDatabaseCommon.cpp.o
#0 28.53 [ 11%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/LightweightDatabaseServer.cpp.o
#0 28.93 [ 11%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/LinuxStrings.cpp.o
#0 28.96 [ 11%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/LogCommandParser.cpp.o
#0 29.07 [ 11%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/MessageFilter.cpp.o
#0 29.25 [ 11%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/NatPunchthrough.cpp.o
#0 29.66 [ 12%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/NetworkIDManager.cpp.o
#0 29.76 [ 12%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/NetworkIDObject.cpp.o
#0 29.88 [ 12%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/PacketConsoleLogger.cpp.o
#0 29.97 [ 12%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/PacketFileLogger.cpp.o
#0 30.02 [ 12%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/PacketLogger.cpp.o
#0 30.26 [ 12%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/PluginInterface.cpp.o
#0 30.30 [ 13%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RPCMap.cpp.o
#0 30.38 [ 13%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakMemoryOverride.cpp.o
#0 30.42 [ 13%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakNetCommandParser.cpp.o
#0 30.65 [ 13%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakNetStatistics.cpp.o
#0 30.84 [ 13%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakNetTransport.cpp.o
#0 31.06 [ 13%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakNetTypes.cpp.o
#0 31.12 /build/thirdparty/raknet/Source/RakNetTypes.cpp: In member function 'void SystemAddress::SetBinaryAddress(const char*)':
#0 31.12 /build/thirdparty/raknet/Source/RakNetTypes.cpp:132:30: warning: writing 1 byte into a region of size 0 [-Wstringop-overflow=]
#0 31.12   132 |                 IPPart[index]=0;
#0 31.12       |                 ~~~~~~~~~~~~~^~
#0 31.12 /build/thirdparty/raknet/Source/RakNetTypes.cpp:123:22: note: at offset 22 into destination object 'IPPart' of size 22
#0 31.12   123 |                 char IPPart[22];
#0 31.12       |                      ^~~~~~
#0 31.15 [ 14%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakNetworkFactory.cpp.o
#0 31.39 [ 14%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakPeer.cpp.o
#0 31.60 /build/thirdparty/raknet/Source/RakPeer.cpp: In function 'void* UpdateNetworkLoop(void*)':
#0 31.60 /build/thirdparty/raknet/Source/RakPeer.cpp:5312:142: note: '#pragma message: -- RakNet: _RAKNET_THREADSAFE not defined.  Do NOT use multiple threads on the same instance of RakPeer (Fast!). --'
#0 31.60  5312 |         #pragma message("-- RakNet: _RAKNET_THREADSAFE not defined.  Do NOT use multiple threads on the same instance of RakPeer (Fast!). --")
#0 31.60       |
                         ^
#0 31.60 /build/thirdparty/raknet/Source/RakPeer.cpp:5333:169: note: '#pragma message: -- RakNet: Using Sleep(). Uncomment USE_WAIT_FOR_MULTIPLE_EVENTS in RakNetDefines.h if you want to use WaitForSingleObject instead. --'
#0 31.60  5333 |                 #pragma message("-- RakNet: Using Sleep(). Uncomment USE_WAIT_FOR_MULTIPLE_EVENTS in RakNetDefines.h if you want to use WaitForSingleObject instead. --")
#0 31.60       |
                                                    ^
#0 32.92 [ 14%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakSleep.cpp.o
#0 32.95 [ 14%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakString.cpp.o
#0 33.26 [ 14%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/RakThread.cpp.o
#0 33.29 [ 14%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/Rand.cpp.o
#0 33.35 [ 15%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ReadyEvent.cpp.o
#0 33.66 [ 15%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ReliabilityLayer.cpp.o
#0 34.58 [ 15%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ReplicaManager.cpp.o
#0 35.13 [ 15%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ReplicaManager2.cpp.o
#0 36.00 [ 15%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/Router.cpp.o
#0 36.70 [ 15%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/SHA1.cpp.o
#0 36.88 [ 16%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/SimpleMutex.cpp.o
#0 36.93 [ 16%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/SocketLayer.cpp.o
#0 37.02 [ 16%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/StringCompressor.cpp.o
#0 37.26 [ 16%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/StringTable.cpp.o
#0 37.47 [ 16%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/SuperFastHash.cpp.o
#0 37.53 [ 16%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/SystemAddressList.cpp.o
#0 37.75 [ 17%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/TCPInterface.cpp.o
#0 38.01 [ 17%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/TableSerializer.cpp.o
#0 38.27 [ 17%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/TelnetTransport.cpp.o
#0 38.40 [ 17%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/ThreadsafePacketLogger.cpp.o
#0 38.46 [ 17%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/_FindFirst.cpp.o
#0 38.54 [ 17%] Building CXX object thirdparty/CMakeFiles/raknet.dir/raknet/Source/rijndael.cpp.o
#0 38.82 [ 18%] Linking CXX static library ../libraknet.a
#0 38.88 [ 18%] Built target raknet
#0 38.88 Scanning dependencies of target mariadb_connector_cpp
#0 38.89 [ 18%] Creating directories for 'mariadb_connector_cpp'
#0 38.97 [ 18%] No download step for 'mariadb_connector_cpp'
#0 38.99 [ 19%] No update step for 'mariadb_connector_cpp'
#0 39.02 [ 19%] No patch step for 'mariadb_connector_cpp'
#0 39.04 [ 19%] Performing configure step for 'mariadb_connector_cpp'
#0 39.11 -- The CXX compiler identification is GNU 11.3.0
#0 39.16 -- The C compiler identification is GNU 11.3.0
#0 39.17 -- Detecting CXX compiler ABI info
#0 39.26 -- Detecting CXX compiler ABI info - done
#0 39.27 -- Check for working CXX compiler: /usr/local/bin/c++ - skipped
#0 39.28 -- Detecting CXX compile features
#0 39.28 -- Detecting CXX compile features - done
#0 39.28 -- Detecting C compiler ABI info
#0 39.34 -- Detecting C compiler ABI info - done
#0 39.36 -- Check for working C compiler: /usr/local/bin/gcc - skipped
#0 39.36 -- Detecting C compile features
#0 39.36 -- Detecting C compile features - done
#0 39.36 -- Looking for floor
#0 39.43 -- Looking for floor - not found
#0 39.43 -- Looking for floor in m
#0 39.50 -- Looking for floor in m - found
#0 39.50 -- Found math lib: m
#0 39.51 -- MariaDB Connector ODBC: INSTALL_BINDIR=bin
#0 39.51 -- MariaDB Connector ODBC: INSTALL_LIBDIR=lib/mariadb
#0 39.51 -- MariaDB Connector ODBC: INSTALL_PCDIR=lib/pkgconfig
#0 39.51 -- MariaDB Connector ODBC: INSTALL_INCLUDEDIR=include/mariadb
#0 39.51 -- MariaDB Connector ODBC: INSTALL_DOCDIR=share/doc/
#0 39.51 -- MariaDB Connector ODBC: INSTALL_LICENSEDIR=share/doc/
#0 39.51 -- MariaDB Connector ODBC: INSTALL_PLUGINDIR=plugin
#0 39.51 -- Libraries installation dir: lib
#0 39.51 -- Authentication Plugins installation dir: plugin
#0 39.51 -- Setting default value for WITH_SSL for libmariadb build to ON
#0 39.51 -- Running C/C cmake scripts
#0 39.52 -- Found Git: /usr/bin/git (found version "2.30.2")
#0 39.54 fatal: not a git repository: /build/thirdparty/mariadb-connector-cpp/../../.git/modules/thirdparty/mariadb-connector-cpp
#0 39.55 fatal: not a git repository: /build/thirdparty/mariadb-connector-cpp/../../.git/modules/thirdparty/mariadb-connector-cpp
#0 39.55 CMake Error at cmake/connector_c.cmake:10 (MESSAGE):
#0 39.55   No MariaDB Connector/C! Run
#0 39.55
#0 39.55       git submodule init
#0 39.55       git submodule update
#0 39.55
#0 39.55   Then restart the build.
#0 39.55
#0 39.55 Call Stack (most recent call first):
#0 39.55   CMakeLists.txt:391 (INCLUDE)
#0 39.55
#0 39.55
#0 39.55 -- Configuring incomplete, errors occurred!
#0 39.55 See also "/build/cmake_build/mariadbcpp/src/mariadb_connector_cpp-build/CMakeFiles/CMakeOutput.log".
#0 39.55 See also "/build/cmake_build/mariadbcpp/src/mariadb_connector_cpp-build/CMakeFiles/CMakeError.log".
#0 39.55 make[2]: *** [thirdparty/CMakeFiles/mariadb_connector_cpp.dir/build.make:125: mariadbcpp/src/mariadb_connector_cpp-stamp/mariadb_connector_cpp-configure] Error 1
#0 39.55 make[1]: *** [CMakeFiles/Makefile2:1551: thirdparty/CMakeFiles/mariadb_connector_cpp.dir/all] Error 2
#0 39.55 make: *** [Makefile:114: all] Error 2
------
failed to solve: executor failed running [/bin/sh -c echo "Build server" &&     mkdir -p cmake_build &&     cd cmake_build &&     sed -i -e "s/171022/${BUILD_VERSION}/g" ../CMakeVariables.txt &&     cmake .. -DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE &&     make -j $BUILD_THREADS]: exit code: 2
</details>

--- a/Docker.md
+++ b/Docker.md
@@ -25,8 +25,7 @@
 
 **NOTE #4**: Make sure to run the following in the repo root directory after cloning so submodules are also downloaded.
 ```
-git submodule init
-git submodule update
+git submodule update --init --recursive
 ```
 **NOTE #5**: If DarkflameSetup fails due to not having cdclient.fdb, rename CDClient.fdb (in the same folder) to cdclient.fdb
 

